docs/examples.md

```diff
@@ -5,7 +5,7 @@
 If you do not pass any inputs, it by default builds a ReSpec or Bikeshed document (`index.html` or `index.bs`) and validates the output. It does not deploy the built document anywhere.
 
 ```yaml
-# .github/workflows/pr.yml
+# Create a file called .github/workflows/pr-push.yml
```

Comment:
This one is definitely a `pr.yml`. Maybe we should use `workflow.yml` or `spec-prod.yml`?

Reply:
As the name doesn't do anything, we should use something general. Otherwise, it's quite confusing. 

See #64 - we need to pick names that are both clear and general. If one can stack multiple actions into a single file, then we should really just call it `action.yml`.

Reply:
Technically, it's not an action. It's a workflow that uses an action. That's why I suggest `workflow.yml`, but I think it's too generic and hard to search for. So, `spec-prod.yml`?